docs/search/get-started-with-search/search-page/log-message-inspector.md

```diff
@@ -0,0 +1,27 @@
+---
+id: log-message-inspector
+title: Log Message Inspector
+sidebar: Log Message Inspector
+description: You can highlight a time range in the histogram for your search results to filter the search results based on that time range.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By default, when performing Log Search queries, the results are displayed in the **Time** and **Messages** column. You can further drill down the results using [Show and Hide Fields in the Field Browser](/docs/search/get-started-with-search/search-page/field-browser/show-hide-fields-in-field-browser).
+
+If you want to view the results with all the parameter field values, you can select all the parameter fields in the **Hidden Fields** tab, which makes the results page clustered. You can overcome this by using the **Log Message Inspector** option. **Log Message Inspector** dock will display an **INFO/ERROR** to quickly identify the status of each log message using the color-code, raw log message, message time, receipt time, and the parameter values associated with the query. 
+
+<img src={useBaseUrl('img/search/get-started-search/search-page/log-message-inspector.png')} width="400" style={{border: '1px solid black'}} alt="log-message-inspector" />
+
+While viewing parameter values, you can add any parameter of your choice to the query by clicking on the kebab menu and selecting the **Filter selected value** option against the field. This narrows down your query to the same tab and avoids opening a new search page, which occurs when you filter the query using the field browser.<br/><img src={useBaseUrl('img/search/get-started-search/search-page/filter-selected-value.png')} width="400" style={{border: '1px solid black'}} alt="filter-selected-value" />
+
+## Viewing the Log Message Inspector
+
+Follow the below steps to open the **Log Message Inspector** dock and view the results of your query with all the parameter values.
+
+1. Enter your query in the search text box.
+1. In the results page, navigate to the row of your choice.
```

Comment:
I think it should be 2. Similarly for next point it should be 3.

Reply:
Its a Markdown syntax. This will render as 1, 2, and 3 in the published page.